src/bundle/ui-dev/src/modules/universal-discovery/content.meta.preview.module.js

```diff
@@ -94,50 +99,71 @@ const ContentMetaPreview = () => {
             </div>
         );
     };
-    const lastModifiedLabel = Translator.trans(/*@Desc("Last modified")*/ 'meta_preview.last_modified', {}, 'universal_discovery_widget');
-    const creationDateLabel = Translator.trans(/*@Desc("Created")*/ 'meta_preview.creation_date', {}, 'universal_discovery_widget');
-    const translationsLabel = Translator.trans(/*@Desc("Translations")*/ 'meta_preview.translations', {}, 'universal_discovery_widget');
+    const renderMetaPreviewLoadingSpinner = () => {
```

Comment:
Given this is inside `ContentMetaPreview` it can be shortened:
```suggestion
    const renderLoadingSpinner = () => {
```